File: core/src/main/java/com/github/julyss2019/bukkit/voidframework/logging/Level.java
package com.github.julyss2019.bukkit.voidframework.logging;


/**
 * 日志等级
 */
public enum Level {
    DEBUG(0),
    INFO(1),
    WARN(2),
    ERROR(3);

    private final int level;

    Level(int level) {
        this.level = level;
    }

    /**
     * 获取整数等级
     */
    public int getIntLevel() {
        return level;
    }
}

File: api-legacy/src/main/java/com/github/julyss2019/bukkit/voidframework/internal/logger/Level.kt
package com.github.julyss2019.bukkit.voidframework.internal.logger

import org.bukkit.ChatColor

enum class Level(val intLevel: Int, val color: ChatColor) {
    DEBUG(0, ChatColor.GRAY),
    INFO(1, ChatColor.WHITE),
    WARN(2, ChatColor.YELLOW),
    ERROR(3, ChatColor.RED)
}